test/shared_test.go

```diff
@@ -223,13 +224,22 @@ func packObjects(tc util.TestCtx, objects map[string]expectedObject) []byte {
 	contentWriter := db.NewTarWriter()
 	defer contentWriter.Close()
 
-	for path, info := range objects {
+	var keys []string
+	for key := range objects {
+		keys = append(keys, key)
+	}
+
+	sort.Strings(keys)
```

Comment:
Damn, this was kind of a consequence of two things:

- Reproducing `packObjects` as a test util instead of using the same function we do in prod
- The fact that the prod version of `packObjects` expects a sorted stream and we use a `map` for convenience in tests

The prod sorting happens here: https://github.com/gadget-inc/dateilager/blob/main/internal/db/tar.go#L233